### ISSUE TITLE
Installation failure on FreeBSD 13

### ISSUE THREAD

**Issue by jpereira** [saltstack/salt-bootstrap#1841]
### Description of Issue/Question

```
pkg: No packages available to install matching 'py38-salt' have been found in the repositories
 * ERROR: Failed to run install_freebsd_stable()!!!
``` 

### Steps to Reproduce Issue

```
./bootstrap.sh -c /tmp -M stable
```

**Comment by myii** [comment]
This should be fixed by #1842, could you confirm and close @jpereira?

**Comment by jpereira** [comment]
I've tried the HEAD version and it looks fixed. thanks
